### PR TITLE
[MRI Violations] Edit protocol table - fix glitchy dialog

### PR DESCRIPTION
**Pull request by davidblader**
https://redmine.cbrain.mcgill.ca/issues/12000
Replaces glitchy jQueryUI dialog box with sweet alert confirmation when user is editing a cell in the mri-protocol table

### CODE REVIEW

**Comment by driusan** [comment]
Changing the library that something is built with is probably a big enough change that it should be in a minor release.

File: modules/mri_violations/js/mri_protocol_violations.js [hunk 1]
```diff
@@ -22,41 +22,52 @@ function save() {
...
+    $('.description').blur(
+      function(event) {
+          event.stopImmediatePropagation();
+          id = event.target.id;
+          value = $('#'+id).text();
+          if (value !== default_value) {
+            $('.description').attr('contenteditable', false);
+            swal({
+              title: "Are you sure?",
+              text: "Are you sure want to edit this field?",
```
Comment (xlecours) [inline modules/mri_violations/js/mri_protocol_violations.js]: typo? 

> Are you sure [you] want...

**Comment by zaliqarosli** [comment]
my link to the table doesnt work .. could this need rebasing? i can access it with the url shown in the screenshot attached to the redmine ticket

**Comment by davidblader** [comment]
> my link to the table doesnt work .. could this need rebasing? i can access it with the url shown in the screenshot attached to the redmine ticket

@zaliqarosli i rebased, can you check again?

**Comment by zaliqarosli** [comment]
@davidblader when i click on the link under the 'Problem' header of the table on the mri violations page, the link takes me to this page instead of the edit protocol table. do i need to change my permissions or something?
screenshot from 2018-02-26 16-32-26

**Comment by zaliqarosli** [comment]
@davidblader i tried to test this again but my front-end doesn't seem to be configured in the right way. maybe best for someone with a project to check it out?

**Pull request merged**